aten/src/ATen/native/native_functions.yaml

```diff
@@ -754,16 +754,12 @@
   dispatch:
     CPU: _log1p__cpu
     CUDA: _log1p__cuda
-    SparseCPU: log1p_sparse_
```

Comment:
why are these removed? O:

Reply:
oops, that's strange... I will revert it